server-kotlin/src/city_search/CitySearchIndex.kt

```diff
@@ -83,12 +84,12 @@ class CitySearchIndex: UntypedAbstractActor() {
     }
 
     fun createCache() {
-        // Read sky to geo mapper
-        // once this file is loaded from s3 (where it's auto updated) we can keep local copy
-        // in resources as a fallback for offline development
-        val dataJson = this::class.java.classLoader.getResource("citylistmetadata_finalresult.json")!!.readText()
+        /*
+            Read sky to geo mapper
+         */
+        val dataJson = File("/../../citylistmetadata_finalresult.json")
```

Comment:
Isn't this breaking change? file still seems to be in resources: https://github.com/HonzaV/cityvizor/blob/feature/CitySynchronization/server-kotlin/resources/citylistmetadata_finalresult.json

Reply:
Yes it is, sorry, will commit original state - I had to comment it out because it wont build the app otherwise

Reply:
fixed

Reply:
👍 thanks